app/javascript/vue/shared/access_modal/edit.vue

```diff
@@ -7,8 +7,22 @@
       <div>
         {{ i18n.t('access_permissions.everyone_else', { team_name: params.object.team }) }}
       </div>
-      <i class="sn-icon sn-icon-info"
-          :title='this.autoAssignedUsers.map((ua) => ua.attributes.user.name).join("\u000d")'></i>
+      <GeneralDropdown @open="loadUsers" @close="closeFlyout">
+        <template v-slot:field>
+          <i class="sn-icon sn-icon-info"
+             :title='this.autoAssignedUsers.map((ua) => ua.attributes.user.name).join("\u000d")'></i>
```

Comment:
i think you can remove title, before it was done just because we didn't have dropdown